userprofile/models.py

```diff
@@ -35,24 +35,21 @@ class Profile(models.Model):
     image = models.ImageField(default="default.jpg", upload_to="profile_pics")
     bocce = models.BooleanField(default=False)
     frisbee = models.BooleanField(default=False)
-    tBall = models.BooleanField(default=False)
-    adultBaseball = models.BooleanField(default=False)
-    adultFootball = models.BooleanField(default=False)
-    adultSoftball = models.BooleanField(default=False)
+    baseball = models.BooleanField(default=False)
+    football = models.BooleanField(default=False)
+    softball = models.BooleanField(default=False)
     basketball = models.BooleanField(default=False)
     cricket = models.BooleanField(default=False)
     flagFootball = models.BooleanField(default=False)
     handball = models.BooleanField(default=False)
     hockey = models.BooleanField(default=False)
     kickball = models.BooleanField(default=False)
     lacrosse = models.BooleanField(default=False)
-    littleLeagueBaseball = models.BooleanField(default=False)
-    littleLeagueSoftball = models.BooleanField(default=False)
     netball = models.BooleanField(default=False)
     rugby = models.BooleanField(default=False)
     tennis = models.BooleanField(default=False)
     volleyball = models.BooleanField(default=False)
-    youthFootball = models.BooleanField(default=False)
+    track = models.BooleanField(default=False)
```

Comment:
Wondering the same thing as above - do we want to add track although there is no map for it?

Reply:
I wasn't sure about that either, I was modeling it after the facilities views.py file which still contains the track field option, if there's data we could just as easily create a map for it. Might be worth bringing it up in standup

` sports = {
         'Bocce': currentFacility['bocce'],
         'Track': currentFacility["track_and"],
         'Frisbee': currentFacility["frisbee"],
         'Baseball': currentFacility["adult_base"],
         'Football': currentFacility["adult_foot"],`